src/webgl/p5.RendererGL.js

```diff
@@ -1701,6 +1715,14 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     return tex;
   }
 
+  // used in imageLight
+  // create a new texture from the img input
+  getBlurryTexture(input){
+    const blurrytex = new p5.Texture(this, input);
```

Comment:
I think the main difference between the blurry textures we store here and the textures already used by RendererGL is that for normal textures, we just make a new p5.Texture directly from the input. For this though, the input is non-blurry, and we want to store the blurry version, so somewhere (maybe in here actually?) we should run the shader to generate the blurry image onto a graphic and store that instead.